recipes/newrelic/apm/java/linux.yml

```diff
@@ -242,19 +275,30 @@ install:
 
           attach_to_pid() {
             javaPid=$1
+            appServerType=$2
             license={{.NEW_RELIC_LICENSE_KEY}}
             region={{.NEW_RELIC_REGION}}
-            appName=DEFAULT_APP_NAME
 
-            echo $(build_stdout_json "Attaching agent to Tomcat process with PID $javaPid...")
+            defaultTomcatAppName=DEFAULT_TOMCAT_APP_NAME
+            defaultJbossAppName=DEFAULT_JBOSS_APP_NAME
+            appServerEchoName="Tomcat"
+
+            if [[ "$appServerType" == "TOMCAT" ]]; then
+              appName=$defaultTomcatAppName
```

Comment:
@mbruzina given we default to tomcat up above, should we just have a if statement for jboss and set the appServerEchoName and appName when not defaulting to tomcat?

Reply:
We can do something different, yes.  This does seem kinda goofy.